### PR TITLE
Mixmix/program cli

### PR DESCRIPTION
**Pull request by mixmix**
Expose all the program methods to programatic CLI.
I expect a few things to change
- interested in feedback on the CLI command signatures ... not sure about some of them
- feel like `src/program/main.ts` signatures ... not sure
- some things gonna change again with new SDK release

## Testing

- [x] Unit tests added/updated
- [ ] Integration tests added/updated
- [ ] Manual testing performed

## Checklist

- [x] I have performed a self-review of my code.
- [ ] I have added tests.
- [x] I have commented my code.
- [ ] I have included a `CHANGELOG.md` entry.
- [ ] I have updated documentation in `github.com:entropyxyz/entropy-docs`, where necessary.

### CODE REVIEW

**Comment by mixmix** [comment]
yo @frankiebee can you have a look at this?
I think lets priortize the mega branch, and loop back to this

**Comment by mixmix** [comment]
@rh0delta it's end of day, I have updated this but not re-reviewed it closely (expect bugs)
If you have a chance can you have a look and see what you think

**Comment by mixmix** [comment]
Depends on: https://github.com/entropyxyz/cli/pull/270

File: src/program/main.ts [hunk 1]
```diff
@@ -59,8 +69,8 @@ export class EntropyProgram extends EntropyBase {
   }
 
   async listDeployed () {
+    // QUESTION: does the CLI ever want to take in an address?
```
Comment (rh0delta) [inline src/program/main.ts:72]: we might want to provide a way for users to check programs owned by any of the keys/addresses attached to their account, but not be able to see owned programs by any address maybe?

**Review by rh0delta — APPROVED** [review]

**Pull request merged**